client/task-list/tasks/payments/components/PaymentMethodList/PaymentMethodList.js

```diff
@@ -65,15 +67,21 @@ export const PaymentMethodList = ( {
 						<CardMedia isBorderless>{ before }</CardMedia>
 						<div className="woocommerce-task-payment__description">
 							{ showRecommendedRibbon && <RecommendedRibbon /> }
-							<H className="woocommerce-task-payment__title">
-								{ title }
-							</H>
+							<Text
+								as="h3"
+								className="woocommerce-task-payment__title"
+							>
+								{ title }{ ' ' }
```

Comment:
Is the `{ ' ' }` just for spacing?  Could we use margin on `SetupRequired` instead?

Reply:
Good catch. Not intentional, I think that's added by prettier if I accidentally leave a trailing space. Already have margin, so just removed this in 46f7d183db1180c6feda6a86168906c9edf3e342.